### ISSUE TITLE
parse error of wikilinks ?

### ISSUE THREAD

**Issue by jlongo** [dendronhq/dendron#3179]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [x] Publish.  External Site Publish
- [x] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

Possible problem in wikulinks to different vault

if i use: 
```
[[some text |dendron://vault/note]]
```
it will work ok

if i use
```
[[some text | dendron://vault/note]]
```
with space in front of pipe it doens't work.

## To Reproduce

see screenshot

## Screenshots

![2022-07-02 16_51_27-Clipboard](https://user-images.githubusercontent.com/1572402/177007528-f8d705c5-9767-49e1-ac82-4f6b0cae7c2c.png)

## Desktop (please complete the following information)

n/a

## Additional context

n/a

## Dendron Log file

n/a

**Comment by kevinslin** [comment]
thanks for the issue! have a fix in pr!

**Comment by kevinslin** [comment]
now fixed in release 0.103
